.github/workflows/self_hosted_e2e.yaml

```diff
@@ -61,14 +61,20 @@ jobs:
           go-version: ${{ env.GOVER }}
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
-      - name: Install docker - MacOS
-        if: matrix.os == 'macos-10.15'
+      - name: Install podman - MacOS
+        if: matrix.os == 'macos-latest'
         run: |
-          brew install --cask docker
-          sudo /Applications/Docker.app/Contents/MacOS/Docker --unattended --install-privileged-components
-          open -a /Applications/Docker.app --args --unattended --accept-license
-          while ! /Applications/Docker.app/Contents/Resources/bin/docker info &>/dev/null; do sleep 1; done
-          docker --version
+          sudo rm -rf `brew --cache`
```

Comment:
just wondering if we should also run podman tests in linux/windows?
@pravinpushkar @shubham1172 thoughts?

Reply:
We can run it for Windows to test the full suite (instead of just slim).

Reply:
I still believe running slim is also good ... we could matrix on both slim and full installs ...

Reply:
> just wondering if we should also run podman tests in linux/windows? @pravinpushkar @shubham1172 thoughts?

Let me check if we can run windows full suite using podman.

> we could matrix on both slim and full installs

If we are able to run full suite the do we need to run for slim mode separately ?

Reply:
Would like to make sure that slim install works with the limitations that we say it works with. We do have to add actor e2e but still have E2E using slim install is something that should do IMO.

Reply:
Yes, slim mode has some extra tests, so it will be good to test both.

Reply:
Enabled slim and complete install matrix for linux and macos.

Reply:
@mukundansundar @shubham1172 Tried installing the podman on windows but could not make it work, see install podman step in windows workflow here - https://github.com/pravinpushkar/cli/runs/7957959381?check_suite_focus=true